test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/Directives/ChunkInheritanceUtilityTest.cs

```diff
@@ -32,83 +32,82 @@ @inherits MyBaseType
                 new UsingChunk { Namespace = "AppNamespace.Model" },
             };
             var cache = new DefaultChunkTreeCache(fileProvider);
-            using (var host = new MvcRazorHost(cache, new TagHelperDescriptorResolver(designTime: false)))
-            {
-                var utility = new ChunkInheritanceUtility(host, cache, defaultChunks);
+            var host = new MvcRazorHost(cache, new TagHelperDescriptorResolver(designTime: false));
 
-                // Act
-                var chunkTreeResults = utility.GetInheritedChunkTreeResults(
-                    PlatformNormalizer.NormalizePath(@"Views\home\Index.cshtml"));
+            var utility = new ChunkInheritanceUtility(host, cache, defaultChunks);
 
-                // Assert
-                Assert.Collection(chunkTreeResults,
```

Comment:
All of this is just untabbed. The diff is a bit messed up. The split view helps a bit